modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -112,6 +114,7 @@ The *Manage Your Verification Methods* interface appears.
 If your organization requires multi-factor authentication and you have removed all verification methods, you must configure at least one verification method before you save your changes.
 . Click *Save*.
 
+[[disable-org-mfa]]
 == Disable Multi-Factor Authentication Requirement for All Organization Users
 
 When you disable the multi-factor authentication requirement, existing users no longer use it to log in to Anypoint Platform and new users log in without being prompted to configure it. Note that disabling multi-factor authentication requirements does not automatically remove multi-factor authentication from user accounts. Individual users who have independently enabled multi-factor authentication must manually remove it in their profile settings.
```

Comment:
I think it is outdated. You cannot disable MFA. You can make it either optional or mandatory